packages/website/ts/components/dropdowns/developers_drop_down.tsx

```diff
@@ -0,0 +1,188 @@
+import { colors } from '@0xproject/react-shared';
+import * as _ from 'lodash';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { DropDown } from 'ts/components/ui/drop_down';
+import { Deco, Key, WebsitePaths } from 'ts/types';
+import { constants } from 'ts/utils/constants';
+import { Translate } from 'ts/utils/translate';
+
+interface KeyToLinkInfo {
+    [key: string]: LinkInfo;
+}
```

Comment:
FYI added an `ObjectMap` type to `types.ts` since this comes up a lot. Not sure why typescript doesn't have a native `Dict` type or something.

Reply:
Nice! We  should adopt this convention everywhere. @LogvinovLeon thoughts?